dotnet/src/Functions/Functions.OpenApi/Model/RestApiSpecification.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// The REST API specification.
+/// </summary>
+public sealed class RestApiSpecification
+{
+    /// <summary>
+    /// The REST API information.
+    /// </summary>
+    public RestApiInfo Info { get; init; }
```

Comment:
nit: Since both Info and Operations are required parameters to the constructor anyway, we can probably change from init to private set.